src/main/java/net/fabricmc/loader/api/FabricLoader.java

```diff
@@ -88,6 +88,19 @@ static FabricLoader getInstance() {
 	 * where the {@code default} adapter is the {@linkplain LanguageAdapter adapter}
 	 * offered by Fabric Loader. </p>
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

Reply:
Uh there is no misnamed class or field? We can't change `LanguageAdapter` to another name without breaking every single mod that uses kotlin and scala.

Reply:
@aimiebell who are you and why are you spamming repositories? Automated codestyle critique without any understanding of project context is not appreciated.

![image](https://user-images.githubusercontent.com/629092/107324521-10b91580-6a5d-11eb-85c1-7d8d2b1b61b5.png)

Reply:
@zml2008 don't bother talking to them they're definitely a bot

Reply:
Spam. Resolving this.